modules/job/sqoop/config/sqoop.xml

```diff
@@ -27,6 +27,7 @@
 				<value>fs.defaultFS=${fsUri}</value>
 				<value>yarn.resourcemanager.address=${resourceManagerHost}:${resourceManagerPort}</value>
 				<value>yarn.application.classpath=${spring.hadoop.yarnApplicationClasspath}</value>
+				<value>mapreduce.framework.name=yarn</value>
```

Comment:
We are only supporting yarn ATM, so we don't need to have that configurable. It's easy to do once there is a need.